go.mod

```diff
@@ -3,9 +3,10 @@ module github.com/cccteam/httpio
 go 1.20
 
 require (
-	github.com/go-chi/chi/v5 v5.0.8
+	github.com/go-chi/chi/v5 v5.0.10
 	github.com/go-playground/errors/v5 v5.3.0
+	github.com/gofrs/uuid v4.4.0+incompatible
```

Comment:
Can we use the google UUID package instead of this one? https://github.com/google/uuid

Reply:
I feel that `github.com/gofrs/uuid` is better maintained and in broader use in the community. I have always used gofrs, which is in the Loan Consolidation package.

We could easily add support for both here or decide which one we want to use.

What is attractive about the google package?